evm-ds/log4rs-local.yml

```diff
@@ -1,17 +1,26 @@
 appenders:
-  stderr:
+  stdout:
     kind: console
-    target: stderr
+    target: stdout
+    encoder:
+      kind: json
```

Comment:
Why do we need json in the "local" configuration?

Reply:
We are seeing this in the console output, and I'm not sure json is the right format for that.